database/seeds.sql

```diff
@@ -0,0 +1,16 @@
+USE toshare;
+
+INSERT INTO family (name)
+VALUES ('Dupont');
+
+INSERT INTO event 
+(
+    date_start,
+    date_end,
+    family_id
+) 
+VALUES (
+    '2019-12-06 15:00:00',
+    '2019-12-06 16:00:00',
+    1
```

Comment:
Care about this foreign key, it might not lead to the expected value as 'Dupont' 's id will be generated automatically